src/pages/learn/txduration.mdx

```diff
@@ -0,0 +1,115 @@
+are coming to consensus on the state of its parachains rather than blocks. In this
+design [validators can vote on the most recent block they think is valid and subsequently
+all the block’s ancestors are implicitly assumed to be valid as well in the vote](https://medium.com/polkadot-network/grandpa-block-finality-in-polkadot-an-introduction-part-1-d08a24a021b5).
+The algorithm then determines the best block which has a >⅔ supermajority of votes
+from validators, and produces a proof-of-finality. The proof-of-finality is constructed
+by taking the supermajority votes, and bundling them up together into a single message.
+In other words, when 2/3 of validators attest to a chain containing a certain block,
+all blocks leading up to that one are finalized at once.{" "}
+
+### Fast Finality (BNB)
+
+BNB was previously using probabilistic finality but it recently implemented fast
+finality in August of 2023 in the [BEP 126](https://forum.bnbchain.org/t/bep-126-draft-introduce-fast-finality-mechanism/123)
+upgrade. Pre fast-finality Binance considered ⅔ _ validatorNum + 1 blocks to be finalized
+(15 blocks). Now after the upgrade it considers (⅔ _ validatorSize) or more of the
+validators vote as expected. This takes about 2.5 blocks as opposed to the 15 blocks
+before the upgrade. This translates into a 2.5 second finality time (down from 7.5
+seconds).
+
+### Fraud Proof Rollup Finality (Arbitrum)
+
+There are different types of finality when dealing with Arbitrum. Once a sequencer
+receives a transaction it will publish a message through its real-time feed confirming
+that it has processed the transaction. This counts as a soft finality of the transaction,
+meaning that the result of that transaction will not change from there on, as long
+as the Sequencer is trustworthy. For a hard finality you would wait for the Sequencer
+to post the batch on L1, although that may take a few minutes (in contrast with the
+soft finality, which is almost instant).
+
+### Casper Friendly Finality Gadget (Eth P.o.S)
+
+Post-merge Ethereum, which no longer uses proof of work uses checkpoint finality.
+The first block at the beginning of each [epoch](https://info.etherscan.com/epoch-in-ethereum/)
+(6.4 minute interval) is a checkpoint. Validators then vote on pairs of checkpoints
+that they consider valid. If the pair of checkpoints receives two-thirds of the staked
+eth vote then the more recent of the two checkpoints gets upgraded to justified.
+When the next pair of epochs is getting justified the previous pair gets upgraded
+to finalized. At the time of this writing this process takes about 15 minutes.
+
+## Common Finality Time For Interchain Transactions
+
+The following is a list of the typical wait times required for finalization on each blockchain connected to Axelar. Each blockchain’s wait time is a combination of the blockchain’s required time to finality plus the time Axelar requires to confirm that blockchain’s finality.
```

Comment:
`wait time` is a new term you are introducing, maybe use one of the standards like `time to confirmation` or another specific step?